vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSteps.java

```diff
@@ -365,14 +371,18 @@ public void assertValueByJsonPath(String json, String jsonPath, String compariso
         getDataByJsonPathSafely(json, jsonPath, true).ifPresent(jsonByPath -> {
             if (jsonByPath.isEmpty())
             {
-                validateEqualsComparison(comparisonRule, null, expectedData);
-                assertThat(jsonPath, null, equalTo(expectedData));
+                validateComparisonRuleAndNullValue(jsonPath, comparisonRule, null, expectedData);
                 return;
             }
             Object jsonValue = jsonByPath.get();
+            if (expectedData == null)
+            {
+                validateComparisonRuleAndNullValue(jsonPath, comparisonRule, jsonValue, null);
+                return;
+            }
```

Comment:
why is it needed?

Reply:
For possibility to execute step using "is not equal to":
```
Given I initialize story variable `json` with value `
{
    "persons": {
        "nemo": null,
        "nemo1": 2
    }
}
`
Then JSON element value from `${json}` by JSON path `$.persons. nemo1` is not equal to `#{null}`
```